lua/nvim_comment.lua

```diff
@@ -137,23 +143,21 @@ end
 function M.setup(user_opts)
   M.config = vim.tbl_extend('force', M.config, user_opts or {})
 
-  -- Messy, change with nvim_exec once merged
-  vim.api.nvim_command('let g:loaded_text_objects_plugin = 1')
-  local vim_func = [[
-  function! CommentOperator(type) abort
-    let reg_save = @@
-    execute "lua require('nvim_comment').operator('" . a:type. "')"
-    let @@ = reg_save
-  endfunction
-  ]]
-  vim.api.nvim_call_function("execute", {vim_func})
-  vim.api.nvim_command("command! -range CommentToggle lua require('nvim_comment').comment_toggle(<line1>, <line2>)")
+  vim.g.loaded_text_objects_plugin = 1
+  api.nvim_exec([[
```

Comment:
This isn't 0.5 safe... FYI why it's done in this backwards way..
Whilst neovim 0.5 is now released it's not available on all distro's so changing this may be a breaking change for some users, best not to put this change in just yet.

Reply:
This still holds IMO

Reply:
Reverted. Although not sure how important pre 0.5 compatibility is. You can just require 0.5 as a minimum.

Reply:
one of the things I wanted was support for <0.5 as selfishly I can use this one some work servers ;P